doc/source/whatsnew/v0.24.0.txt

```diff
@@ -112,7 +112,7 @@ Timezones
 Offsets
 ^^^^^^^
 
--
+- :class:`Tick` subclasses can no longer be created with the argument `normalize=True` (:issue:`????`)
```

Comment:
In cases like these, reference the PR itself (unless an issue is created as pointed out <a href="https://github.com/pandas-dev/pandas/pull/21427#discussion_r194555863">here</a>).

Reply:
The same goes for the test case (i.e. add a comment referencing the PR).